pages/common/dotnet.md

```diff
@@ -0,0 +1,19 @@
+# dotnet
+
+> Cross Platform .NET Command Line Tools for .NET Core
+
+- Initialize a new .NET project:
+
+`dotnet new project-type`
+
+- Restore nuget packages:
```

Comment:
Maybe good to explain what does restore do actually.

Reply:
Restore is a nuget-specific term that basically means "make all the installed packages equal those defined in the nuget configuration file" - which is basically like `npm install`.

Reply:
Yes, so I'm guessing dotnet users will be familiar with nuget packages.